src/Dfe.PlanTech.Web/Views/Shared/NotFoundError.cshtml

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-grid-row">
```

Comment:
Could this go in Contentful?

Reply:
If I remember right we decided not to do that because this page could be the result of contentful being down, I reckon we should leave it local